deploy/helm/charts/templates/cluster-exporter.yaml

```diff
@@ -45,4 +45,16 @@ spec:
             - name: METRICS_LISTEN_PORT
               value: :{{ .Values.ndmExporter.clusterExporter.metricsPort }}
             {{- end }}
-{{- end }}
\ No newline at end of file
+{{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 6 }}
```

Comment:
why do we need to add a new line here but not for the rest of the indents?

Reply:
We add the new line everywhere. Or did I misunderstood the question.

Reply:
I mean the new line through `nindent`. For others we are using `indent`

Reply:
I'd rather this was nindent 8 to keep it consistent with existing list fields.

Reply:
updated